pgactivity/data.py

```diff
@@ -165,7 +165,12 @@ def pg_is_local_access(self) -> bool:
             with open(pid_file, "r") as fd:
                 pid = fd.readline().strip()
         except PermissionError:
-            logger.info("Access Denied to the pidfile. System counters are disabled.")
+            logger.info(
+                f"Access Denied to the pidfile: {pid_file}. System counters are disabled."
+            )
+            return False
+        except FileNotFoundError:
+            logger.info(f"pidfile not found: {pid_file}. System counters are disabled.")
```

Comment:
We could catch `OSError` and log the specific error:
```python
except OSError as e:
     logger.info("pidfile %s could not be read: %s. System counters are disabled.", pid_file, e)
    return False
```

Also note that we should not format the log message ourselves as it possibly does superfluous work if the logging level is higher than `INFO` here.

Reply:
> Also note that we should not format the log message ourselves as it possibly does superfluous work if the logging level is higher than INFO here.

... clever ... :)